WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -139,12 +141,19 @@ final class HubMenuViewModel: ObservableObject {
         observeSiteForUIUpdates()
         observePlanName()
         tapToPayBadgePromotionChecker.$shouldShowTapToPayBadges.share().assign(to: &$shouldShowNewFeatureBadgeOnPayments)
+        createCardPresentPaymentService()
     }
 
     func viewDidAppear() {
         NotificationCenter.default.post(name: .hubMenuViewDidAppear, object: nil)
     }
 
+    func createCardPresentPaymentService() {
```

Comment:
nit: could be private

Updated in https://github.com/woocommerce/woocommerce-ios/pull/12970/commits/e4f821057c54a13f7439c02fa814574b6683e867.

```suggestion
    private func createCardPresentPaymentService() {
```